### ISSUE TITLE
Remove already stopped container

### ISSUE THREAD

**Issue by bit0rez** [DockStation/dockstation#39]
Hi, guys! :)
I found a little bug.
On "Containers" tab, if i trying to remove started container, it removed successful. But, if i try to remove stopped container, i get an error from application:
```
    Container remove failed
    (HTTP code 304) container already stopped -
```
Ubuntu Linux 16.04 LTS
DockStation v1.2.2

**Comment by igor-lemon** [comment]
Yep. We missed the bug. Thank you for issue.

**Comment by igor-lemon** [comment]
Fixed v1.2.3
